### ISSUE TITLE
Show branch labels

### ISSUE THREAD

**Issue by trvrb** [nextstrain/auspice#174]
There is an control toggle to "show branch labels". This currently sets `this.state.showBranchLabels` in redux but it has no impact on app behavior. This should function like it does in v1, where branches are labeled with mutations with some decisions on depth of information to display.

<img width="693" alt="branch-labels" src="https://cloud.githubusercontent.com/assets/1176109/22678202/3933b3c6-ecad-11e6-9550-dda0a29ab581.png">

The code for this in v1 is here:

* https://github.com/blab/nextflu/blob/master/auspice/js/auspice/tree.js#L129
* https://github.com/blab/nextflu/blob/master/auspice/js/auspice/tree.js#L167
* https://github.com/blab/nextflu/blob/master/auspice/js/auspice/tree.js#L226

**Comment by rneher** [comment]
@jameshadfield I am getting an error when toggling the "showBranchLabels". `this.props.dispatch` is not a function. surely something trivial, but this whole redux/radium is still vodoo to me.

**Comment by jameshadfield** [comment]
Yeah - definately some voodoo! You need to use `connect` which is the decorator that links in redux state (as you choose) and dispatch into any component that needs it. So change the start of `toggle-branch-labels.js` to:
```
import React from "react";
import Radium from "radium";
import { TOGGLE_BRANCH_LABELS } from "../../actions/types";
import {dataFont, darkGrey} from "../../globalStyles";
import { connect } from "react-redux";

@Radium
@connect()
class ToggleBranchLabels extends React.Component {
```

**Comment by rneher** [comment]
thanks! but why does `@Radium` has no (), but `@connect()` does?

**Comment by trvrb** [comment]
This was resolved by @rneher in #186.
